integration_tests/env.go

```diff
@@ -122,20 +127,33 @@ func resolveEnvVars(d Defaults, t *testing.T) Vars {
 		t.Logf("%s env var not set, using default", BucketVar)
 		b = d.BucketDefault
 	}
-	t.Logf("Using bucekt %s", b)
+	t.Logf("Using bucket %s", b)
 
 	r := os.Getenv(BucketRegionVar)
 	if r == "" {
 		t.Logf("%s env var not set, using default", d.BucketRegionDefault)
 		r = d.BucketRegionDefault
 	}
 	t.Logf("Using bucekt region %s", r)
+
+	a := os.Getenv(BucketAccessKeyIdVar)
+	if r == "" {
+		t.Logf("%s env var not set, using default", d.BucketRegionDefault)
+		r = d.BucketRegionDefault
+	}
+	s := os.Getenv(BucketSecretAccessKeyVar)
+	if r == "" {
+		t.Logf("%s env var not set, using default", d.BucketRegionDefault)
```

Comment:
I believe this log message needs to be updated with the secret access key var?

I think both of these checks need to fail as well. Presumably if we don't have permissions into AWS then we can't run our integration tests?

Reply:
Yeah, you are right we need to have the AWS credentials in order to make tests work, will mark those as a `Faltaf`